### DISCUSSION TITLE
Managing update and create timestamp in DynamoDB

### DISCUSSION THREAD

**Discussion by dtrce** [awslabs/aws-sdk-rust#908] [Q&A]
Hello,

I'm using the put_item to perform create or update of the rows. I have 2 attributes created and updated which is supposed to hold the timestamps of when the record was created/update. How can I ensure that when the record is created, it will only set the created timestamp and when it is updated, it will keep created timestamp as is and only update the updated attribute with new timestamp?

**Comment by rcoh** [comment]
Hello! which service is this referring to?

**Reply by dtrce** [reply]
oh sorry, DynamoDB.
